lib/httplog/http_log.rb

```diff
@@ -95,14 +96,33 @@ def log_body(body, encoding = nil, content_type = nil)
 
       if config.prefix_response_lines
         log('Response:')
-        log_data_lines(data)
+        log_data_lines(filter_out(data))
       else
-        log("Response:\n#{data}")
+        log("Response:\n#{filter_out(data)}")
       end
     rescue BodyParsingError => e
       log("Response: #{e.message}")
     end
 
+    def filter_out(msg)
```

Comment:
I'd add a guard clause here so we don't do any of the work below if there aren't any keywords to filter.